__test__/integration/order_api/order_scenario_runner.ts

```diff
@@ -85,13 +100,13 @@ export class OrderScenarioRunner {
                 // the scenario specifies that a signature from a signatory
                 // ought to be attached.
                 debtOrder.debtorSignature = scenario.signatories.debtor
-                    ? await this.orderSigner.asDebtor(debtOrder)
+                    ? await this.orderSigner.asDebtor(debtOrder, false)
                     : undefined;
```

Comment:
just for the record, and this could just be me, I'm used to seeing `null` used in this kind of situation with ternaries, rather than `undefined.`

Reply:
I'm not strongly opinionated either way.  Is there a reason to use null instead?

Reply:
just for ref; when assigning assign to `null` - https://codeburst.io/javascript-null-vs-undefined-20f955215a2